### PR TITLE
add a sample backup and rollback task to use on production routers

### PR DESCRIPTION
**Pull request by zahodi**
This is useful if you are running these modules in production as it will create a backup of your mikrotik device prior and schedule a rollback if you accidentally break something with ansible. 

Placing the sample task in the tasks/ folder until we figure out where to best place our yml.

### CODE REVIEW

**Comment by zahodi** [comment]
@senorsmile can you review?

**Comment by senorsmile** [comment]
How do we envision people using this sample?  Just copy/pasting into their real playbooks?

File: tasks/backup_recover.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      - sensitive
+      - test
+      - read
+      - policy
+
+###################################################################
+# Place this in the end of your mikrotik playbook this will remove the
+# rollback scheduler task if the playbook runs succesfully
+#####################################################################
+- name: remove rollback scheduler task if run succesfull
```
Comment (senorsmile) [inline tasks/backup_recover.yml]: succesSful

File: tasks/backup_recover.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+- name: run command module to create a backup
+  mt_command:
+    hostname: "{{ mt_hostname }}"
+    username: "{{ mt_user }}"
+    password: "{{ mt_pass }}"
+    command: /system/backup/save
+    command_arguments:
+      name: pre_deploy
+      password: 123
+- name: add rollback scheduler task
```
Comment (senorsmile) [inline tasks/backup_recover.yml]: add blank line above for legibility please :)

File: tasks/backup_recover.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+################################################################
+# TEMPORARY TASK TO STORE THE ROLLBACK PROCEDURE
```
Comment (senorsmile) [inline tasks/backup_recover.yml]: this is a temporary PLAYBOOK, not TASK... technically this playbook contains a list of tasks.

**Review by senorsmile — CHANGES_REQUESTED** [review]
just a few typos

**Pull request merged**